webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -1102,7 +1106,7 @@ public function verifyAction(
             $redirect = $this->generateUrl('jury_submission_by_judging', ['jid' => $judgingId]);
         }
 
-        return $this->redirect($redirect);
+        return $this->redirectToLocalReferrer($this->router, $request, $redirect);
```

Comment:
Old line 1096/new line 1100 above has a comment that describes the logic that follows, i think with this change the described logic/distinction is undone?

Reply:
Same here